sleap/gui/dialogs/frame_range.py

```
@@ -0,0 +1,42 @@
+            min_frame_idx_field.setValue(1)
+
+            max_frame_idx_field.setRange(1, max_frame_idx)
+            max_frame_idx_field.setValue(max_frame_idx)
+
+        min_frame_idx_field.valueChanged.connect(self._update_max_frame_range)
+        max_frame_idx_field.valueChanged.connect(self._update_min_frame_range)
+
+        self.setWindowTitle(title)
+
+    def _update_max_frame_range(self, value):
+        min_frame_idx_field = self.form_widget.fields["min_frame_idx"]
```

Ruff (0.8.2)

Local variable `min_frame_idx_field` is assigned to but never used

Remove assignment to unused variable `min_frame_idx_field`

(F841)